snapcraft/tests/store/test_store_client.py

```diff
@@ -373,7 +387,21 @@ def test_get_account_information_refreshes_macaroon(self):
                         'private': False,
                         'price': None,
                         'since': '2016-12-12T01:01:01Z',
-                    }
+                    },
+                    'test-snap-with-dev': {
+                        'price': None,
+                        'private': False,
+                        'since': '2016-12-12T01:01:01Z',
+                        'snap-id': 'test-snap-id-with-dev',
+                        'status': 'Approved'
+                    },
+                    'test-snap-with-no-validations': {
+                        'price': None,
+                        'private': False,
+                        'since': '2016-12-12T01:01:01Z',
+                        'snap-id': 'test-snap-id-with-no-validations',
+                        'status': 'Approved'
+                    },
```

Comment:
I know. But the thing is that in unit tests we shouldn't just check the reply from a fake server. We can maybe check that the store client returns a json, and a couple of fields, not all the details of what we hardcoded in the fake.